docs/guides/run_a_job.md

```diff
@@ -0,0 +1,261 @@
+!!! important "Quick Look"
+    `floyd run [OPTIONS] [COMMAND]`
+
+    **[[OPTIONS]](#options_1):**
+
+    - [Instance Type](#instance-type): `--cpu` **or** `--gpu` **or** `--cpu+` **or** `--gpu+`
```

Comment:
This doesn't seem to be rendered well on the UI

Reply:
Thanks for pointing this out. I fixed this and the fix is live now